Sources/Receipt/ListReceiptViewPresenter.swift

```diff
@@ -89,18 +119,46 @@ final class ListReceiptViewPresenter {
                     strongSelf.isLoadInProgress = false
                     strongSelf.view.hideLoading()
                     if let error = error {
-                        strongSelf.view.showError(error, { strongSelf.listReceipt() })
+                        strongSelf.view.showError(error, { strongSelf.listUserReceipts() })
                         return
                     } else if let result = result {
                         strongSelf.groupReceiptsByMonth(result.data)
-                        strongSelf.areAllReceiptsLoaded = result.data.count < strongSelf.limit ? true : false
+                        strongSelf.areAllReceiptsLoaded =
+                            result.data.count < strongSelf.userReceiptLimit ? true : false
                         strongSelf.offset += result.data.count
                     }
                     strongSelf.view.loadReceipts()
                 }
             }
     }
 
+    private func listPrepaidCardReceiptHandler()
+        -> (HyperwalletPageList<HyperwalletReceipt>?, HyperwalletErrorType?) -> Void {
+            return { [weak self] (result, error) in
+                guard let strongSelf = self else {
+                    return
+                }
+                DispatchQueue.main.async {
+                    strongSelf.isLoadInProgress = false
+                    strongSelf.view.hideLoading()
+                    if let error = error,
+                        let prepaidCardToken = strongSelf.prepaidCardToken {
+                        strongSelf.view.showError(error, { strongSelf.listPrepaidCardReceipts(prepaidCardToken) })
+                        return
+                    } else if let result = result {
```

Comment:
I see you have not pushed the latest logic of prepaidCardReceipts pagination

Reply:
@jsaini-hw I did